docs/worker/pip-mirror.md

```diff
@@ -0,0 +1,176 @@
+# Installing a Pip Mirror
+
+When running python based lambdas in OpenLambda packages
+are installed with pip. By default, pip installs packages
+through an API to pypi.org, which cannot always be trusted.
+
+Pypi.org cannot guarantee that every indexed package is safe,
+thus a local mirrored index for pip may be beneficial.
```

Comment:
Maybe you can describe two advantages: the performance advantage and the security advantage (an allowlist).